cmd/main.go

```diff
@@ -255,7 +256,7 @@ func loadConfigs() {
 	}
 
 	// anything we need to set based on our main Numaplane Config?
-	numaLogger.SetLevel(config.LogLevel)
+	numaLogger.SetLevel(logger.LogLevelMap[strings.ToUpper(config.LogLevel)])
```

Comment:
so, if somebody has a string that isn't one of the values, the result of `logger.LogLevelMap[strings.ToUpper(config.LogLevel)]` is 0, and it looks like `numaLogger.SetLevel()` will set to default log level if that happens. Maybe we can at least log an error if that happens?

Reply:
Added [here](https://github.com/numaproj/numaplane/pull/557/files#diff-72d1bf15c143c98293b1321f13f60e3500c88a4cca40aed1e0bf93c4a165ae3fR268)